pkg/tsdb/cloudwatch/constants/metrics.go

```diff
@@ -357,7 +357,7 @@ var NamespaceMetricsMap = map[string][]string{
essBridgeSourceTSSyncLoss", "EgressBridgeSourceTotalPackets", "EgressBridgeSourceTransportError", "FailoverSwitches", "IngressBridgeBitRate", "IngressBridgeCATError", "IngressBridgeCRCError", "IngressBridgeContinuityCounter", "IngressBridgeDroppedPackets", "IngressBridgeFailoverSwitches", "IngressBridgeMergeActive", "IngressBridgeNotRecoveredPackets", "IngressBridgePATError", "IngressBridgePCRAccuracyError", "IngressBridgePCRError", "IngressBridgePIDError", "IngressBridgePMTError", "IngressBridgePTSError", "IngressBridgePacketLossPercent", "IngressBridgeRecoveredPackets", "IngressBridgeTSByteError", "IngressBridgeTSSyncLoss", "IngressBridgeTotalPackets", "IngressBridgeTransportError", "IngressBridgeSourceARQRecovered", "IngressBridgeSourceARQRequests", "IngressBridgeSourceBitRate", "IngressBridgeSourceCATError", "IngressBridgeSourceCRCError", "IngressBridgeSourceContinuityCounter", "IngressBridgeSourceDroppedPackets", "IngressBridgeSourceFECPackets", "IngressBridgeSourceFECRecovered", "IngressBridgeSourceMergeActive", "IngressBridgeSourceMergeLatency", "IngressBridgeSourceNotRecoveredPackets", "IngressBridgeSourceOverflowPackets", "IngressBridgeSourcePATError", "IngressBridgeSourcePCRAccuracyError", "IngressBridgeSourcePCRError", "IngressBridgeSourcePIDError", "IngressBridgeSourcePMTError", "IngressBridgeSourcePTSError", "IngressBridgeSourcePacketLossPercent", "IngressBridgeSourceRecoveredPackets", "IngressBridgeSourceRoundTripTime", "IngressBridgeSourceTSByteError", "IngressBridgeSourceTSSyncLoss", "IngressBridgeSourceTotalPackets", "IngressBridgeSourceTransportError", "Jitter", "Latency", "MergeActive", "MergeLatency", "MaintenanceScheduled", "MaintenanceRescheduled", "MaintenanceCanceled", "MaintenanceCanceled", "MaintenanceStarted", "MaintenanceSucceeded", "MaintenanceFailed", "NotRecoveredPackets", "OutputConnected", "OutputDisconnections", "OutputTotalBytes", "OutputDroppedPayloads", "OutputLatePayloads", "OutputTotalPayloads", "OverflowPackets", "PATError", "PCRAccuracyError", "PCRError", "PIDError", "PMTError", "PTSError", "PacketLossPercent", "RecoveredPackets", "RoundTripTime", "SourceARQRecovered", "SourceARQRequests", "SourceBitRate", "SourceCATError", "SourceCRCError", "SourceConnected", "SourceContinuityCounter", "SourceDisconnections", "SourceDroppedPackets", "SourceFECPackets", "SourceFECRecovered", "SourceMergeActive", "SourceSelected", "SourceMergeLatency", "SourceMergeStatusWarnMismatch", "SourceMergeStatusWarnSolo", "SourceNotRecoveredPackets", "SourceMissingPackets", "SourceOverflowPackets", "SourcePATError", "SourcePCRAccuracyError", "SourcePCRError", "SourcePIDError", "SourcePMTError", "SourcePTSError", "SourcePacketLossPercent", "SourceRecoveredPackets", "SourceRoundTripTime", "SourceTSByteError", "SourceTSSyncLoss", "SourceTotalPackets", "SourceTotalBytes", "SourceDroppedPayloads", "SourceLatePayloads", "SourceTotalPayloads", "SourceTransportError", "TSByteError", "TSSyncLoss", "TotalPackets", "TransportError", "Uptime"},
```

Comment:
Looks good, but just fixing some of the alphabetization
```suggestion
	"AWS/MediaConnect":         {"ARQRecovered", "ARQRequests", "BitRate", "CATError", "Connected", "ConnectedOutputs", "ContinuityCounter", "ConsecutiveDrops", "ConsecutiveNotRecovered", "ConnectionAttempts", "CRCError", "Disconnections", "DroppedPackets", "EgressBridgeBitRate", "EgressBridgeCATError", "EgressBridgeContinuityCounter", "EgressBridgeCRCError", "EgressBridgeDroppedPackets", "EgressBridgeFailoverSwitches", "EgressBridgeMergeActive", "EgressBridgeNotRecoveredPackets", "EgressBridgePacketLossPercent", "EgressBridgePATError", "EgressBridgePCRAccuracyError", "EgressBridgePCRError", "EgressBridgePIDError", "EgressBridgePMTError", "EgressBridgePTSError", "EgressBridgeRecoveredPackets", "EgressBridgeSourceBitRate", "EgressBridgeSourceCATError", "EgressBridgeSourceCRCError", "EgressBridgeSourceContinuityCounter", "EgressBridgeSourceDroppedPackets", "EgressBridgeSourceMergeActive", "EgressBridgeSourceMergeLatency", "EgressBridgeSourceNotRecoveredPackets", "EgressBridgeSourcePacketLossPercent", "EgressBridgeSourcePATError", "EgressBridgeSourcePCRAccuracyError", "EgressBridgeSourcePCRError", "EgressBridgeSourcePIDError", "EgressBridgeSourcePMTError", "EgressBridgeSourcePTSError", "EgressBridgeSourceRecoveredPackets", "EgressBridgeSourceTotalPackets", "EgressBridgeSourceTransportError", "EgressBridgeSourceTSByteError", "EgressBridgeSourceTSSyncLoss", "EgressBridgeTotalPackets", "EgressBridgeTransportError", "EgressBridgeTSByteError", "EgressBridgeTSSyncLoss",  "FailoverSwitches", "IngressBridgeBitRate", "IngressBridgeCATError", "IngressBridgeContinuityCounter", "IngressBridgeCRCError", "IngressBridgeDroppedPackets", "IngressBridgeFailoverSwitches", "IngressBridgeMergeActive", "IngressBridgeNotRecoveredPackets", "IngressBridgePacketLossPercent", "IngressBridgePATError", "IngressBridgePCRAccuracyError", "IngressBridgePCRError", "IngressBridgePIDError", "IngressBridgePMTError", "IngressBridgePTSError", "IngressBridgeRecoveredPackets", "IngressBridgeSourceARQRecovered", "IngressBridgeSourceARQRequests", "IngressBridgeSourceBitRate", "IngressBridgeSourceCATError", "IngressBridgeSourceCRCError", "IngressBridgeSourceContinuityCounter", "IngressBridgeSourceDroppedPackets", "IngressBridgeSourceFECPackets", "IngressBridgeSourceFECRecovered", "IngressBridgeSourceMergeActive", "IngressBridgeSourceMergeLatency", "IngressBridgeSourceNotRecoveredPackets", "IngressBridgeSourceOverflowPackets", "IngressBridgeSourcePATError", "IngressBridgeSourcePacketLossPercent", "IngressBridgeSourcePCRAccuracyError", "IngressBridgeSourcePCRError", "IngressBridgeSourcePIDError", "IngressBridgeSourcePMTError", "IngressBridgeSourcePTSError", "IngressBridgeSourceRecoveredPackets", "IngressBridgeSourceRoundTripTime","IngressBridgeSourceTotalPackets", "IngressBridgeSourceTransportError",  "IngressBridgeSourceTSByteError", "IngressBridgeSourceTSSyncLoss", "IngressBridgeTotalPackets", "IngressBridgeTransportError", "IngressBridgeTSByteError", "IngressBridgeTSSyncLoss", "Jitter", "Latency", "MaintenanceCanceled", "MaintenanceRescheduled", "MaintenanceScheduled", "MaintenanceStarted", "MaintenanceSucceeded", "MaintenanceFailed", "MergeActive", "MergeLatency", "NotRecoveredPackets", "OutputConnected", "OutputDisconnections", "OutputDroppedPayloads", "OutputLatePayloads", "OutputTotalBytes", "OutputTotalPayloads", "OverflowPackets", "PacketLossPercent", "PATError", "PCRAccuracyError", "PCRError", "PIDError", "PMTError", "PTSError", "RecoveredPackets", "RoundTripTime", "SourceARQRecovered", "SourceARQRequests", "SourceBitRate", "SourceCATError", "SourceConnected", "SourceContinuityCounter", "SourceCRCError", "SourceDisconnections", "SourceDroppedPackets", "SourceDroppedPayloads",  "SourceFECPackets", "SourceFECRecovered", "SourceLatePayloads", "SourceMergeActive", "SourceMergeLatency", "SourceMergeStatusWarnMismatch", "SourceMergeStatusWarnSolo", "SourceMissingPackets", "SourceNotRecoveredPackets", "SourceOverflowPackets", "SourcePacketLossPercent", "SourcePATError", "SourcePCRAccuracyError", "SourcePCRError", "SourcePIDError", "SourcePMTError", "SourcePTSError", "SourceRecoveredPackets", "SourceRoundTripTime", "SourceSelected", "SourceTotalBytes", "SourceTotalPayloads", "SourceTotalPackets", "SourceTransportError", "SourceTSByteError", "SourceTSSyncLoss", "TotalPackets", "TransportError", "TSByteError", "TSSyncLoss", "Uptime"},
```

Reply:
I fixed the alphabetization also.

Reply:
okay done, sorted the list.